type-definitions/dom-event-test-flow.js

```diff
@@ -0,0 +1,10 @@
+// @flow
```

Comment:
We've been using the `type-definitions` folder as the canonical spot for TypeScript files.  We've been keeping flow tests in `test/flow` ([for example](https://github.com/mostjs/core/tree/master/packages/core/test/flow)).

What do you think about moving this file to `test/flow/dom-event-test.js`?

Reply:
Okay. But you already accepted this in most itself
https://github.com/cujojs/most/tree/master/type-definitions

Reply:
Ooops, yes, I must not have been paying attention in that particular case.  Sorry about that.  I'll move them in cujojs/most as well.